### ISSUE TITLE
Okhttp needs update for Android 11

### ISSUE THREAD

**Issue by agardes** [NativeScript/canvas#32]
As mentionned on [this nativescript issue](https://github.com/NativeScript/NativeScript/issues/9356) I reported, I had a crash while testing my app on Android 11 API 30 : 

```
StackTrace:
java.lang.ExceptionInInitializerError
	at okhttp3.internal.platform.Platform.get(Platform.java:85)
	at okhttp3.OkHttpClient.newSslSocketFactory(OkHttpClient.java:263)
	at okhttp3.OkHttpClient.<init>(OkHttpClient.java:229)
	at okhttp3.OkHttpClient.<init>(OkHttpClient.java:202)
	at io.socket.engineio.client.Socket.<init>(Socket.java:215)
	at io.socket.engineio.client.Socket.<init>(Socket.java:174)
	at io.socket.client.Manager$Engine.<init>(Manager.java:638)
	at io.socket.client.Manager$1.run(Manager.java:277)
	at io.socket.thread.EventThread.exec(EventThread.java:55)
	at io.socket.client.Manager.open(Manager.java:266)
	at io.socket.client.Manager.open(Manager.java:256)
	at io.socket.client.Socket$3.run(Socket.java:142)
	at io.socket.thread.EventThread$2.run(EventThread.java:80)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)
Caused by: java.lang.IllegalStateException: Expected Android API level 21+ but was 30
	at okhttp3.internal.platform.AndroidPlatform.buildIfSupported(AndroidPlatform.java:238)
	at okhttp3.internal.platform.Platform.findPlatform(Platform.java:202)
	at okhttp3.internal.platform.Platform.<clinit>(Platform.java:79)
	... 16 more
```

Managed to fix it by adding `dependencies { implementation 'com.squareup.okhttp3:okhttp:4.9.0' }` to the app.gradle,

Apparently the plug-in canvas-three is causing the crash.

**Comment by triniwiz** [comment]
You can try cleaning the project (ns clean) build again and lmk

**Comment by yschimke** [comment]
Also fixed in 3.14.7 and 3.12.13 of OkHttp

**Comment by agardes** [comment]
@triniwiz @yschimke thanks !
